app/static/js/script.js

```diff
@@ -0,0 +1,268 @@
+		<section id="my-modal">
+			<div class="heading modal">
+				<h1>${portfolioCard.title[0]}</h1>
+				<span class="close">&times;</span>
+			</div>
+			<ul class="works-badges">
+				${renderLanguagesLi(portfolioCard.badge1)}
+			</ul>
+			<img src="${modalWindow.image}" alt="background image">
+			<div class="bottom-container">
+				<div class="para">
+					<p>${modalWindow.modalPara}</p>
+				</div>
+				<div class="technologies">
+					<ul class="tech-list modal">
+						${renderLanguagesLi(modalWindow.technologies)}
+					</ul>
+					<div class="demo-btns">
+					<ul class="buttons">
+						${renderLanguagesLi(modalWindow.buttons)}
+					<ul>
+					</div>
+				</div>
+			</div>
+		</section>
+	</div>
+`;
+
+const mobileMod = `
+	<section id="my-mobile-modal">
+		<div class="heading modal">
+			<h1>${portfolioCard.title[0]}</h1>
+			<span class="close-mobile">&times;</span>
+		</div>
+		<ul class="works-badges mobile modal">
+			${renderLanguagesLi(portfolioCard.badge1)}
+		</ul>
+		<img src="${modalWindow.mobileImage}" alt="background image">
+		<div class="bottoms-container">
+			<div class="mob-para modal">
+				<p>${modalWindow.mobilePara}</p>
+			</div>
+			<div class="mob-tech">
+				<ul class="tech-list mobile modal">${renderLanguagesLi(portfolioCard.language1)}</ul>
+				</ul>
+				<img src="${modalWindow.lineImg}" alt="seperator image">
+				<div class="demo-btns">
+				<ul class="buttons">
+					${renderLanguagesLi(modalWindow.buttons)}
+				<ul>
+				</div>
+			</div>
+		</div>
+	</section>
+`
+
+
+
+// Desktop
+document.getElementById("card1").insertAdjacentHTML('beforeend', cardOne);
+document.getElementById("card2").insertAdjacentHTML("beforeend", cardTwo);
+document.getElementById("card3").insertAdjacentHTML("beforeend", cardThree);
+document.getElementById("card4").insertAdjacentHTML("beforeend", cardFour);
+// Mobile
+document.getElementById('mobile1').insertAdjacentHTML("beforeend", cardOneMobile);
+document.getElementById("mobile2").insertAdjacentHTML("beforeend", cardFourMobile);
+document.getElementById("mobile3").insertAdjacentHTML("beforeend", cardThreeMobile);
+document.getElementById("mobile4").insertAdjacentHTML("beforeend", cardTwoMobile);
+
+document.body.innerHTML += mobileMod;
+document.body.innerHTML += modals;
+
+
+
+// Modal close
+function close(){
+	document.getElementById("my-modal").style.display = 'none';
+}
+
+function closeMobile(){
+	document.getElementById("my-mobile-modal").style.display = 'none';
+}
+
+// Modal load
+function loadModalPopup(){
+	let span = document.getElementsByClassName("close")[0];
+	if (document.body.clientWidth > 768){
+		document.getElementById("my-modal").style.display = 'block';
+		span.addEventListener("click", close);
+	} else if (document.body.clientWidth <= 768) {
+        let spann = document.getElementsByClassName("close-mobile")[0];
+		document.getElementById("my-mobile-modal").style.display = 'block';
+		spann.addEventListener("click", closeMobile);
+	}	
+}
```

Comment:
- [ ] [REQUIRED] The `modal` opens automatically when you `start` your project and `refresh` the web page, please fix this problem because the `modal` should only open after clicking the button!